packages/mongodb-dialects/java-driver/src/main/kotlin/com/mongodb/jbplugin/dialects/javadriver/glossary/PsiMdbTreeUtil.kt

```diff
@@ -270,24 +270,30 @@ fun PsiElement.findMongoDbCollectionReference(): PsiExpression? {
 /**
  * Resolves to the value of the expression if it can be known at compile time
  * or null if it can only be known at runtime.
```

Comment:
We should change the JavaDoc here to avoid misleading usage. I'm thinking something along the lines:

```suggestion
 * Resolves to a pair of the resolved value in the expression and whether it was possible to
 * resolve the value or not.
```